spec/lib/krikri/mapper_agent_spec.rb

```diff
@@ -1,11 +1,50 @@
+require 'spec_helper'
+
 describe Krikri::Mapper::Agent do
-  opts = { name: :agent_map, generator_uri: 'http://example.org/agent/gen/1' }
-  it_behaves_like 'a software agent', opts
+  before(:all) do
+    DatabaseCleaner.clean_with(:truncation)
+    create(:krikri_mapping_activity)
+    create(:krikri_harvest_activity)
+  end
+
+  harvest_gen_uri_str = \
```

Comment:
Same comment as for [`aggregation_entity_behavior_spec.rb`](https://github.com/dpla/KriKri/pull/131/files#r27392504) for bot here and `mapping_gen_uri_str`, without @no-reply's feedback.